packages/mobile/src/verify/VerificationInputScreen.tsx

```diff
@@ -168,6 +98,8 @@ class VerificationInputScreen extends React.Component<Props, State> {
       }
       this.setState({ timer: timer - 1 })
     }, 1000)
+    this.keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', this.onKeyboardShow)
+    this.keyboardDidHideListener = Keyboard.addListener('keyboardDidHide', this.onKeyboardHide)
```

Comment:
If you end up needing `<KeyboardSpacer />` on this screen for iOS, it has a `onToggle` prop that could simplify this a little.

Reply:
Cool, I'll test iOS today and see if that's needed/helpful!